src/librbd/api/Mirror.h

```diff
@@ -82,6 +90,15 @@ struct Mirror {
   static void image_get_info(ImageCtxT *ictx,
                              mirror_image_info_t *mirror_image_info,
                              Context *on_finish);
+  static int image_get_info(librados::IoCtx& io_ctx,
```

Comment:
Nit: since this doesn't seem to be used, can we drop it for now?

Reply:
Again, it is already merged to the master. And it is used internally in Mirror.cc. Certainly we could live without it, just using the async version in Mirror.cc, though personally I like as it is now. But I can push a PR (or add a commit to this PR) to drop it, if you still think it is a good idea.

Reply:
If it's used, it's fine.